agent/eni/gonetwrapper/generate_mocks_windows.go

```diff
@@ -0,0 +1,16 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package gonetwrapper
```

Comment:
Can't this be netwrapper? Why do you need to prefix it with go

Reply:
This is done following the application specific naming convention.
For example, a wrapper over netutils is named netutile wrapper and a wrapper over dev is named udevwrapper.
Since, this package is a wrapper over Golang's net utils, we are naming it gonetwrapper.

Also, network related utilities are available in networkutils package and naming this package netutils may lead to additional confusion.